server.js

```diff
@@ -1,42 +1,59 @@
-const express = require('express')
+const express = require("express")
 const app = express()
-//const usersSeed = require('./public/pirates.json')
-let db
 
-app.use(express.static('public'))
+app.use(express.static("public"))
 
 const html = `
 <!doctype html>
-<html class="no-js" lang="">
+<html class="no-js" lang="fr">
   <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <title>Pronostics World Cup</title>
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
   </head>
   <body>
-    <div id="bloc_page">
+    <a href='/matchs/new'>New Match</a>
+    <div id="main">
     </div>
     <script src="page.js"></script>
     <script src="app.js"></script>
   </body>
 </html>`
 
-/*
-app.post('/pirates', (req, res) => {
-  return insertUser(req.body)
-  .then(record => res.json(record))
+app.get("/matchs", (req, res) => {
+  //console.log("matchs powa")
+  const matchs = [
+    {
+      teamHome: "France",
+      teamOut: "Egypte",
+      scoreTeamHome: 10,
+      scoreTeamOut: 1,
+      hours: "20:45",
+      localisation: "Moscou"
+    },
+    {
+      teamHome: "France",
+      teamOut: "Egypte",
+      scoreTeamHome: 10,
+      scoreTeamOut: 1,
+      hours: "20h40",
+      localisation: "Moscou"
+    },
+  ]
+  res.json(matchs)
 })
 
-app.get('/pirates', (req, res) => {
-  db.all('SELECT * from users')
-  .then(records => res.json(records))
-})
-*/
+// app.get("/matchs/new",(req, res) => {
```

Comment:
delete errors don't keep them in comment unless you add a special comment "never do this"